tardis/io/model_reader.py

```diff
@@ -85,19 +85,66 @@ def read_abundances_file(abundance_filename, abundance_filetype,
     """
 
     file_parsers = {'simple_ascii': read_simple_ascii_abundances,
-                    'artis': read_simple_ascii_abundances}
+                    'artis': read_simple_ascii_abundances,
+                    'isotopes': read_simple_isotope_abundances}
```

Comment:
I would actually call this tardis_model. I think this can be some sort of default model. where you specify things in this way